ansible/roles/test/tasks/acltb_ranges_test.yml

```diff
@@ -0,0 +1,112 @@
+# Usage example:
+#   ansible-playbook test_sonic.yml -i inventory --limit arc-switch1028 --tags acltb_ranges_test -b -e ranges_number=3 -e ptf_host=ptf-fib
+#
+# Variable "ranges_number" defines how many ACL range objects the test will attempt to create
+#
+
+# Input arguments check
+- fail: msg="ranges_number is not defined."
+  when: ranges_number is not defined
+
+# Set facts for ACL configuration
+- set_fact:
+    acltb_configs:
+      - "{{ 'acl_ranges_table' }}"
+      - "{{ 'acl_ranges_rules' }}"
+
+# Set facts for the loganalizer
+- set_fact:
+    testname: acl
+    run_dir: /tmp
+    out_dir: /tmp/ansible-loganalyzer-results
+    test_match_file: acltb_match_messages.txt
+    test_ignore_file: acltb_ignore_messages.txt
+    test_expect_file: acltb_expect_messages.txt
+    match_file: loganalyzer_common_match.txt
+    ignore_file: loganalyzer_common_ignore.txt
+
+# Separate set_fact is required to be able to use 'testname' fact.
+- set_fact:
+    testname_unique: "{{ testname }}.{{ ansible_date_time.date}}.{{ ansible_date_time.hour}}-{{ ansible_date_time.minute}}-{{ ansible_date_time.second}}"
+
+# Separate set_fact is required to be able to use 'testname_unique' fact.
+- set_fact:
+    test_out_dir: "{{ out_dir }}/{{testname_unique}}"
+    match_file_list: "{{ run_dir }}/{{test_match_file}},{{ run_dir }}/{{match_file}}"
+    ignore_file_list: "{{ run_dir }}/{{test_ignore_file}},{{ run_dir }}/{{ignore_file}}"
+    result_file: result.loganalysis.{{testname_unique}}.log
+    summary_file: summary.loganalysis.{{testname_unique}}.log
+
+# Gather minigraph facts
+- name: Gathering minigraph facts about the device
+  minigraph_facts: host={{ inventory_hostname }}
+  become: no
+  connection: local
+
+- name: Read port reverse alias mapping
+  set_fact:
+    alias_reverse_map: "{{ lookup('file', 'roles/sonicv2/files/ssw/{{ sonic_hwsku }}/alias_reverse_map.json') | from_json }}"
+
```

Comment:
not sure...
I just used the solution used in fib, lag, everflow and other tests